### PR TITLE
[Cleanup] Catalog -> StorageManager in Codegen

### PR DESCRIPTION
**Pull request by seojungmin**
This PR is a part of the clean-up's, and modifies `catalog` to `storage_manager` in codegen, which is more correct of naming.

### CODE REVIEW

**Comment by seojungmin** [comment]
@pmenon this is ready for review.

**Comment by seojungmin** [comment]
I add @tcm-marcel to the review list. Any of you is fine to review. Thanks!

File: src/include/codegen/query.h [hunk 1]
```diff
@@ -55,7 +55,7 @@ class Query {
   // this query.
   bool Prepare(const QueryFunctions &funcs);
 
-  // Execute th e query given the catalog manager and runtime/consumer state
+  // Execute th e query given the storage manager and runtime/consumer state
```
Comment (pmenon) [inline src/include/codegen/query.h]: Fix typo.

**Comment by seojungmin** [comment]
Thanks, @pmenon. This should be merged after its builds pass.

**Comment by apavlo** [comment]
I can merge.

**Pull request merged**